docs/ECLWatch/TheECLWatchMan.xml

```diff
@@ -1138,6 +1138,188 @@
+      role="bold">Operations</emphasis> link, then click on <emphasis
+      role="bold">Dynamic ESDL</emphasis> from the navigation sub-menu
+      bar.</para>
+
+      <para>The DESDL-based ESP services available are listed in the navigator
+      pane on the left as children of their parent ESP process.</para>
+
+      <figure>
+        <title>Dynamic ESDL sub-menu</title>
+
+        <mediaobject>
+          <imageobject>
+            <imagedata fileref="images/DESDL03.jpg" />
+          </imageobject>
+        </mediaobject>
+      </figure>
+
+      <para>This tab contains a list of all DESDL based ESP Services and their
+      ESDL Binding information.. For more information about Dynamic ESDL refer
+      to the documentation; <emphasis>Dynamic ESDL</emphasis> available from
+      the HPCC Systems portal: <ulink
+      url="https://hpccsystems.com/download/documentation/all">https://hpccsystems.com/download/documentation/all</ulink></para>
+
+      <sect2 id="USING_Dynamic_ESDL">
+        <title>Using Dynamic ESDL</title>
+
+        <para>Click on the triangle to expand the ESP process and display the
+        DESDL services. Select the desired DESDL service from the navigator
+        pane.</para>
+
+        <para>The selected service's information displays on the <emphasis
+        role="bold">Summary</emphasis> tab to the right.</para>
+
+        <para>Select the <emphasis role="bold">Binding</emphasis> tab to
+        display the ESDL definition in XML format and the configuration
+        information. Press the Definition button to view the XML.</para>
+
+        <para><figure>
+            <title>Dynamic ESDL Binding Definition</title>
+
+            <mediaobject>
+              <imageobject>
+                <imagedata fileref="images/DESDL02.jpg" />
+              </imageobject>
+            </mediaobject>
+          </figure></para>
+
+        <para>You can select ESP Services and assign them an interface (an
+        ESDL Definition) and configure each available method.</para>
+
+        <sect3 id="Configure_ESDL_Bindings">
+          <title>Configuring ESDL Bindings</title>
+
+          <para>You can select a service and if there is a binding for it you
+          can review, delete, or modify the configuration of that
+          binding.</para>
+
+          <para>Press the <emphasis role="bold">Configuration</emphasis>
+          button to view or add to the configurations. <figure>
+              <title>Dynamic EDSL Binding Configuration</title>
+
+              <mediaobject>
+                <imageobject>
+                  <imagedata fileref="images/DESDL04.jpg" />
+                </imageobject>
+              </mediaobject>
+            </figure></para>
+
+          <para><emphasis role="bold">To Add an attribute:</emphasis></para>
+
+          <para><orderedlist>
+              <listitem>
+                <para>press the arrow next to the method (if there are are
```

Comment:
There's a couple of issues w/ this line: Press should be capitalized, and there seems to be an extra "are" in the parenthesis, but I recommend changing this sentence to something like:
"Expand the target method configuration attributes (if there are.."